### ISSUE TITLE
is it possible to increase java heap space? 

### ISSUE THREAD

**Issue by kingfish777** [bd2kccd/r-causal#14]
Is it possible to increase java heap space from within R? My data consists of 3.6M observations of 20 variables.
Thanks!

Error in .jnew("RectangularArrayBuilder", .jcast(array), dim) : 
  java.lang.OutOfMemoryError: Java heap space

**Comment by kingfish777** [comment]
The trick: 
options( java.parameters = "-Xmx48g" )

**Comment by kingfish777** [comment]
oh wait, I see that there is a java.parameters as a flag in the gfci.discrete api. Trying that.

**Comment by chirayukong** [comment]
Did the java.parameters on gfci.discrete work?

**Comment by kingfish777** [comment]
I don't think so. It should not have run out of memory. I am able to load and run the Mayo Clinic data in Tetrad GUI, but not r-causal. Heap space remains a problem. For whatever reason, I don't think that the java.parameter is working. I tried to set the heap space for the general session as well using option().
